runtime/service/npud/core/org.tizen.npud.xml

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
```

Comment:
org.tizen.* files are for tizen's platform service, right ?
Is there any documentation for those spec ?

Reply:
This spec is came from dbus guideline. This spec is not related to Tizen, but I think we should think about the service name, `org.tizen.npud`.
https://dbus.freedesktop.org/doc/dbus-api-design.html